### PR TITLE
Removes ability from simple mobs to help invoke runes

### PR DESCRIPTION
**Pull request by kugamo**
## What Does This PR Do

Removes ability from simple mobs to help invoke runes.

## Why It's Good For The Game

Simple mobs like mice, slimes, or lizards should not be able to help invoke runes or count towards ascension.

These changes are untested, due to cult gamemode playercount requirement... Merge with caution.

## Changelog
:cl:
fix: Fixed simple mobs being able to contribute to invoking cult runes.
/:cl:

### CODE REVIEW

**Comment by AffectedArc07** [comment]
Question with this, does this impact constructs, since those are also simple mobs.

**Comment by kugamo** [comment]
> Question with this, does this impact constructs, since those are also simple mobs.

Yeah it will, I overlooked this. DNM for now. I will rework this PR

**Comment by Sirryan2002** [comment]
Mice shouldn't be able to help with runs 👍👍👍

File: code/game/gamemodes/cult/runes.dm [hunk 1]
```diff
@@ -360,11 +362,12 @@ structure_check() searches for nearby cultist structures required for the invoca
 			H.uncuff()
 			H.Silence(6 SECONDS) //Prevent "HALP MAINT CULT" before you realise you're converted
 
-			var/obj/item/melee/cultblade/dagger/D = new(get_turf(src))
-			if(H.equip_to_slot_if_possible(D, slot_in_backpack, FALSE, TRUE))
-				to_chat(H, "<span class='cultlarge'>You have a dagger in your backpack. Use it to do [SSticker.cultdat.entity_title1]'s bidding. </span>")
-			else
-				to_chat(H, "<span class='cultlarge'>There is a dagger on the floor. Use it to do [SSticker.cultdat.entity_title1]'s bidding.</span>")
+			if(!isanimal(convertee))
```
Comment (moxian) [inline code/game/gamemodes/cult/runes.dm:365]: There is already an `if(ishuman(convertee))` a bit above (line 342); this `isanimal` check looks redundant and kinda confusing.

**Pull request closed without merging**